pkg/network/usm/monitor_test.go

```diff
@@ -402,9 +402,21 @@ func (s *HTTPTestSuite) TestHTTPMonitorIntegration() {
 		t.Run(tt.name, func(t *testing.T) {
 			for _, keepAliveEnabled := range []bool{true, false} {
 				t.Run(testNameHelper("with keep alive", "without keep alive", keepAliveEnabled), func(t *testing.T) {
-					testHTTPMonitor(t, tt.targetAddress, tt.serverAddress, 100, testutil.Options{
-						EnableKeepAlive: keepAliveEnabled,
-					})
+					monitor := newHTTPMonitor(t)
+
+					srvDoneFn := testutil.HTTPServer(t, tt.serverAddress, testutil.Options{EnableKeepAlive: keepAliveEnabled})
+					t.Cleanup(srvDoneFn)
+
+					// Perform a number of random requests
```

Comment:
lets make the doc more understandable, that the random is the content of the requests.